Test/SqlDom/BackupRestoreRegressionTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System.Collections.Generic;
+using Microsoft.SqlServer.TransactSql.ScriptDom;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SqlStudio.Tests.AssemblyTools.TestCategory;
+
+namespace SqlStudio.Tests.UTSqlScriptDom
+{
+    using System;
+    using System.IO;
```

Comment:
nit: Move these to the top of the file

Reply:
Thanks! - addressed in 2926e5154dc4f8147178f2660b3cf2cd7b99095b.